src/util/numeric.h

```diff
@@ -243,6 +243,23 @@ inline s32 myround(f32 f)
 	return (s32)(f < 0.f ? (f - 0.5f) : (f + 0.5f));
 }
 
+inline constexpr f32 sqr(f32 f)
+{
+	return f * f;
+}
+
+inline f32 ipow(f32 base, u8 exponent)
```

Comment:
Note: this function is not used in this PR anymore, but is nevertheless helpful.